internal/controller/atlasorgsettings/handler.go

```diff
@@ -91,11 +91,7 @@ func (h *AtlasOrgSettingsHandler) unmanage(orgID string) (ctrlstate.Result, erro
 }
 
 func (h *AtlasOrgSettingsHandler) HandleInitial(ctx context.Context, aos *akov2.AtlasOrgSettings) (ctrlstate.Result, error) {
-	return h.upsert(ctx, state.StateInitial, state.StateCreated, aos)
-}
-
-func (h *AtlasOrgSettingsHandler) HandleCreated(ctx context.Context, aos *akov2.AtlasOrgSettings) (ctrlstate.Result, error) {
-	return h.upsert(ctx, state.StateCreated, state.StateUpdated, aos)
+	return result.NextState(state.StateUpdated, "Updated AtlasOrgSettings.")
```

Comment:
nit: please stay consistent with the language. `Updated AtlasOrgSettings` starts with uppercase, other cases start with lower letter, i.e. `unmanaged AtlasOrgSettings for orgID %s.`

Reply:
Thanks! Fixed. All transition messages are now started with a capital letter.